src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1564,9 +1565,12 @@ private static bool NormalizeTaskTypesInPointer(CSharpCompilation compilation, r
             }
             if (type.ContainsNullableReferenceTypes())
             {
-                // PROTOTYPE(NullableReferenceTypes): Create attribute
-                // with PEModuleBuilder.SynthesizeNullableAttribute().
-                // See AttributeTests_Nullable.EmitAttribute_Interface.
+                // PROTOTYPE(NullableReferenceTypes): Not including top-level nullability.
+                var attr = moduleBuilder.SynthesizeNullableAttribute(type, TypeSymbolWithAnnotations.Create(type, isNullableIfReferenceType: null));
```

Comment:
>SynthesizeNullableAttribute [](start = 41, length = 27)

Not related to this PR: I noticed that  `SynthesizeTupleNamesAttribute` and  `SynthesizeDynamicAttribute` are defined in `CSharpCompilation`. 
But  `SynthesizeIsByRefLikeAttribute`, ` SynthesizeIsUnmanagedAttribute` and `SynthesizeIsReadOnlyAttribute` are defined in `PEModuleBuilder`. They are are very simple (no encoding) so just forward to `Compilation.TrySynthesizeAttribute`.

`SynthesizeNullableAttribute` isn't simple. I wonder if it should be moved next to the methods for tuple names and dynamic, in `CSharpCompilation`.

Reply:
Added `// PROTOTYPE` comment.

---
In reply to: [194196765](https://github.com/dotnet/roslyn/pull/27589#discussion_r194196765) [](ancestors = 194196765)